jax/_src/nn/functions.py

```diff
@@ -890,6 +907,7 @@ def dot_product_attention(
     is_causal: bool = False,
     query_seq_lengths: ArrayLike | None = None,
     key_value_seq_lengths: ArrayLike | None = None,
+    local_window_size: tuple[int, int] | None = None,
```

Comment:
What about making this either an int or a tuple of ints? If single int, then equivalent to symmetric window.

Reply:
Flax supports this for eg. convolution kernels.

Reply:
Sure. Done.